Source/dom/assets/__tests__/GradientAsset.test.js

```diff
@@ -20,15 +20,15 @@ test('should create gradient asset from MSGradientAsset', (_context, document) =
 })
 
 test('should create gradient asset with name', (_context, document) => {
-  const gradient = {
-    name: 'My Gradient',
-    gradient: {},
-  }
-
-  document.gradients.push(gradient)
-  expect(document.gradients.length).toEqual(1)
-
-  const asset = document.gradients[0]
-  expect(asset.gradient.type).toBe('Gradient')
-  expect(asset.name).toBe('My Gradient')
+//   const gradient = {
+//     name: 'My Gradient',
+//     gradient: {},
+//   }
+// 
+//   document.gradients.push(gradient)
+//   expect(document.gradients.length).toEqual(1)
+// 
+//   const asset = document.gradients[0]
+//   expect(asset.gradient.type).toBe('Gradient')
+//   expect(asset.name).toBe('My Gradient')
```

Comment:
Are these changes unintentional? I am assuming this test needs fixing but will be addressed later? If that's the case, I'd suggest to leave it as is instead of commenting out.

Reply:
Oh damn yes!  Sorry!

Reply:
The perils of editing in bbedit and committing on the command line